src/components/common/NetworkBanner.js

```diff
@@ -41,16 +43,36 @@ const Container = styled.div`
             display: inline-block;
         }
     }
-`
 
-const Header = styled.h4`
-    margin-bottom: 5px !important;
+    h4 {
+        margin-bottom: 5px !important;
+    }
 `
 
-const NetworkBanner = (props) => {
+const NetworkBanner = ({ account }) => {
+
+    useEffect(() => {
+        setBannerHeight()
+        window.addEventListener("resize", setBannerHeight)
+        return () => {
+            window.removeEventListener("resize", setBannerHeight)
+        }
+    }, [account])
+
+    const setBannerHeight = () => {
+        const bannerHeight = document.getElementById('top-banner') && document.getElementById('top-banner').offsetHeight
+        const app = document.getElementById('app-container')
+        const navContainer = document.getElementById('nav-container')
+        navContainer.style.top = bannerHeight ? `${bannerHeight}px` : 0
+        app.style.paddingTop = bannerHeight ? `${bannerHeight + 85}px` : '75px'
+    }
+
+    const isMainnetGuestFlow = IS_MAINNET && !account.accountId && !window.location.href.includes('staging')
+    const isMainnetStaging = IS_MAINNET && window.location.href.includes('staging')
```

Comment:
@vgrichina do you know if there's a better way to determine if it's staging? Not sure if it would make sense to change the `IS_MAINNET` variable to `CURRENT_NETWORK` or similar.

Reply:
@Patrick1904 we always can just set specific env variable in given environment. E.g. `SHOW_BETA_WARNING` or something like this.

Reply:
@Patrick1904 you can specify it for staging here https://github.com/near/near-wallet/blob/2529bab9ca6a95efd01e8d4a0a0a00c5be6024be/netlify.toml#L5